upstairs/src/lib.rs

```diff
@@ -4290,12 +4257,38 @@ impl Upstairs {
          * Walk the downstairs work active list, and pull out all the active
```

Comment:
Could we start this comment with "To build the dependency list for this flush, we ..."  Reading this I was not sure what we were building here.  I do note that this comment was here before you made these changes, so I understand I'm asking you to improve the comment that was here before you (probably written by me). Thanks!

Reply:
addressed in 4691ec5